editor/icons/source/icon_GUI_tree_arrow_right.svg

```diff
@@ -71,7 +71,7 @@
      id="layer1"
      transform="translate(0,-1040.3622)">
     <path
-       style="fill:none;stroke:#ffffff;stroke-width:2;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:0.39215687"
```

Comment:
why do they need to be #ffffff and can't be #e0e0e0 if I add #FFFFFF to #000000 than there would be another additional colors changed which might should star the same. because #FFFFFF is used really commenly.